lib/Sema/TypeCheckAvailability.h

```diff
@@ -68,6 +69,29 @@ enum class ExportabilityReason : unsigned {
   ExtensionWithConditionalConformances
 };
 
+/// A description of the restrictions on what declarations can be referenced
+/// from a the signature or body of a declaration.
+///
+/// We say a declaration is "exported" if it is `public` or
+/// `@usableFromInline`, not `_@spi`, and not visible via an
```

Comment:
There are typos here `_@spi` and above `from a the signature `.

Reply:
Thanks!